components.json

```
@@ -1039,109 +1039,141 @@
 			"path": "plugins/{id}/prism-{id}",
 			"link": "plugins/{id}/"
 		},
-		"line-highlight": "Line Highlight",
+		"line-highlight": {
+			"title": "Line Highlight",
+			"description": "Highlights specific lines and/or line ranges."
+		},
 		"line-numbers": {
 			"title": "Line Numbers",
+			"description": "Line number at the beginning of code lines.",
 			"owner": "kuba-kubula"
 		},
 		"show-invisibles": {
 			"title": "Show Invisibles",
+			"description": "Show hidden characters such as tabs and line breaks.",
 			"after": [
 				"autolinker",
 				"data-uri-highlight"
 			]
 		},
-		"autolinker": "Autolinker",
-		"wpd": "WebPlatform Docs",
+		"autolinker": {
+			"title": "Autolinker",
+			"description": "Converts URLs and emails in code to clickable links. Parses Markdown links in comments."
+		},
+		"wpd": {
+			"title": "WebPlatform Docs",
+			"description": "Makes tokens link to <a href=\"https://webplatform.github.io/docs/\">WebPlatform.org documentation</a>. The links open in a new tab."
+		},
 		"custom-class": {
 			"title": "Custom Class",
+			"description": "This plugin allows you to prefix Prism default classes (e.g. <code>.comment</code> will become <code>.namespace--comment</code>) or replace them with your defined ones (like <code>.editor__comment</code> or <code>.comment_7sh3a</code>).",
```

For consistency with the others:

```suggestion
			"description": "Allows you to prefix Prism default classes (e.g. <code>.comment</code> will become <code>.namespace--comment</code>) or replace them with your defined ones (like <code>.editor__comment</code> or <code>.comment_7sh3a</code>).",
```